privacyidea/static/contrib/js/ngmodules/escape-text.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
```

Comment:
Is this module self written?

In contrib we only have external modules. So if this is your own code, you need to put it somewhere else like add it to app.js or the contoller of the policies.

Reply:
it is indeed self-written.